site/cds_rdm/inspire_harvester/transform_entry.py

```diff
@@ -146,69 +181,263 @@ def _transform_publication_date(self):
 
     def _transform_document_type(self):
         """Mapping of INSPIRE document type to resource type."""
-        document_type = self.inspire_metadata.get("document_type")[0]
-        document_type_mapping = {
-            "activity report": "publication-report",
-            "article": "publication-article",
-            "book": "publication-book",
-            "book chapter": "publication-section",
-            "conference paper": "publication-conferencepaper",
-            "note": "publication-technicalnote",
-            "proceedings": "publication-conferenceproceeding",
-            "report": "publication-report",
-            "thesis": "publication-thesis",
-        }
+        # document_type = self.inspire_metadata.get("document_type")[0]
+        #
+        # document_type_mapping = {
+        #     "activity report": "publication-report",
+        #     "article": "publication-article",
+        #     "book": "publication-book",
+        #     "book chapter": "publication-section",
+        #     "conference paper": "publication-conferencepaper",
+        #     "note": "publication-technicalnote",
+        #     "proceedings": "publication-conferenceproceeding",
+        #     "report": "publication-report",
+        #     "thesis": "publication-thesis",
+        # }
+        #
+        # if document_type not in document_type_mapping:
+        #     self.metadata_errors.append(
+        #         f"Error occurred while mapping document_type to resource_type. Couldn't fine a mapping rule for "
+        #         f"document_type {document_type}. INSPIRE record id: {self.inspire_metadata.get('control_number')}."
+        #     )
+        #     return None
+
+        # return document_type_mapping.get(document_type)
+
+        document_types = self.inspire_metadata.get("document_type", [])
+        for doc_type in document_types:
+            if doc_type == "article":
+                self.metadata_errors.append("Articles are not supported for now.")
+                return None
+
+        # uncomment the part above for other doc types (thesis have only 1 possible doc type)
```

Comment:
I think we should have a task for this

Reply:
RIght, also not sure why not enabling it now already, in any case I have created the issue: https://github.com/CERNDocumentServer/cds-rdm/issues/498